prov/tcp/src/tcpx.h

```diff
@@ -103,11 +110,40 @@ int tcpx_conn_mgr_init(struct tcpx_fabric *tcpx_fabric);
 
 void tcpx_conn_mgr_close(struct tcpx_fabric *tcpx_fabric);
 
+ssize_t tcpx_comm_send(struct tcpx_pe_entry *pe_entry, const void *buf, size_t len);
+
+ssize_t tcpx_comm_recv(struct tcpx_pe_entry *pe_entry, void *buf, size_t len);
+
+ssize_t tcpx_comm_flush(struct tcpx_pe_entry *pe_entry);
+
 int tcpx_progress_init(struct tcpx_domain *domain, struct tcpx_progress *progress);
 
 int tcpx_progress_close(struct tcpx_domain *domain);
+
```

Comment:
nit: It's easier to read header files if function declarations are grouped closer together (no spacing) when they are logically grouped together.